tests/test-summary-view.sh

```diff
@@ -49,18 +49,18 @@ ${CMD_PREFIX} ostree --repo=repo pull --mirror origin
 assert_has_file repo/summary
 output=$(${OSTREE} summary --view)
 echo "$output" | sed -e 's/^/# /'
-echo "$output" | grep --quiet --no-messages "* main"
-echo "$output" | grep --quiet --no-messages "* other"
-echo "$output" | grep --quiet --no-messages "ostree.summary.last-modified"
-echo "$output" | grep --quiet --no-messages "Static Deltas (ostree.static-deltas): {}"
+echo "$output" | grep --quiet --no-messages --fixed-strings "* main"
+echo "$output" | grep --quiet --no-messages --fixed-strings "* other"
+echo "$output" | grep --quiet --no-messages --fixed-strings "ostree.summary.last-modified"
+echo "$output" | grep --quiet --no-messages --fixed-strings "Static Deltas (ostree.static-deltas): {}"
```

Comment:
I think this would be more nicely written with `libtest-core.sh` like:
```
$OSTREE summary -- view > out.txt
for value in "* main" "* other" ...; do
  assert_file_has_content_literal out.txt $value
done
```

(After adding `assert_file_has_content_literal` which would be a variant of `assert_file_has_content`)